recipe/meta.yaml

```
@@ -60,13 +61,12 @@ requirements:
     - lightgbm ==3.3.5
     - shap ==0.42.1
     - mlflow >=2.1.0,<2.4
```

```suggestion
    - mlflow >=2.1.0,<2.4
    - sentence-transformers>=2.2.2,<3
```
See https://github.com/snowflakedb/snowflake-ml-python/compare/1.3.0...1.3.1#diff-4db974eadd3011661a082fa9471f344e2086c3005084558cbcaf337e39eec304R53

---

Done!